FrejaEidClient/src/main/java/com/verisec/frejaeid/client/enums/RegistrationState.java

```diff
@@ -0,0 +1,55 @@
+package com.verisec.frejaeid.client.enums;
+
+/**
+ *
+ * @author vedrbuk
+ */
+public enum RegistrationState {
```

Comment:
We have this explanation before MinRegLevel class /**
 * Minimum required registration level of a user in order to approve/decline
 * transaction:
 * <br> - {@linkplain #BASIC}
 * <br> - {@linkplain #EXTENDED}
 * <br> - {@linkplain #PLUS}
 *
 */

So maybe to add something for this one, because they look similar?

Reply:
Done.